clients/Admin/src/assets/config/config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
For reference solution do we need this file? We will use this only for workshop right?